src/components/Order_page/Order_card/Order_card.scss

```diff
@@ -0,0 +1,45 @@
+.order_card{
+  background-color: #F8F9FA;
+  width: 100%;
+  padding: 20px;
+  position: fixed;
+  bottom: 0;
+  right: 0;
+
+}
+h3{
+  color: #007bff;
+  padding-bottom: 16px;
+  font-weight: 600;
+}
+.order_costs {
+  li{
+    display: flex;
+    justify-content: space-between;
+    padding-top:5px;
+  }
+}
+
+.total{
+  font-weight: 700;
+}
+.order_btn{
+  background-color: #1DA74C;
+  border-style: none;
+  color: #ffffff;
+  padding: 15px;
+  font-size: 20px;
+  border-radius: 5px;
+  margin-top: 10px;
+  font-weight: 600;
+}
```

Comment:
This should be nested within .order_costs

Reply:
Nested within order card instead